environment/docker-compose.yml

```diff
@@ -45,24 +45,43 @@ services:
         condition: service_healthy
 
   postgres:
-    image: postgres:14
+    build:
+      context: ./postgres
+      dockerfile: Dockerfile
+    restart: always
     environment:
-      POSTGRES_DB: ${POSTGRES_DB}
-      POSTGRES_USER: ${DEFAULT_USER}
-      POSTGRES_PASSWORD: ${DEFAULT_PASSWORD}
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: bichard
+      POSTGRES_USER: bichard
       PGDATA: /var/lib/postgresql/data/pgdata
+    volumes:
+      - ./postgres/schema.sql:/docker-entrypoint-initdb.d/init-db.sql
```

Comment:
We keep the `schema.sql` for creating the Conductor database. The rest of the schema is created by liquibase.